dev/ese/src/_etw/Microsoft-ETW-ESE.mc

```diff
@@ -0,0 +1,2423 @@
+﻿<?xml version='1.0' encoding='utf-8' standalone='yes'?>
+
+<!--
+Copyright (c) Microsoft Corporation.  All rights reserved.
+-->
+<!--
+Use of this source code is subject to the terms of the Microsoft shared
```

Comment:
What is this?  Did you add this?  Clearly we maybe gave this source out before to someone else, maybe Exchange partners or something.

Reply:
Oh I take that back ... this might also be verbiage from like a sample ETW .mc file that we pulled that we never changed.  I think that makes a bit more sense.  I'll let you decide if you want to just delete this.

Reply:
Delete the text, right?